react-facet/docs/api/hooks.md

```diff
@@ -182,19 +226,23 @@ const InputField = ({ placeholderFacet, valueFacet }) => {
 
 Optionally, you can pass an equality check function as the fourth argument to `useFacetMap`. This is particularly useful when grouping more than one facet together into a single array / object, since hard equality checks will not work on arrays / objects.
 
-```tsx
-import { tupleEqualityCheck, useFacetMap } from '@react-facet/dom-fiber'
+```tsx twoslash
+const SubComponent = ({ facets }: { facets: any }) => null
+// ---cut---
+// @esModuleInterop
+// @errors: 2322 1011
+import { shallowArrayEqualityCheck, useFacetState, useFacetMap } from '@react-facet/core'
 
 
-const WrapperComponent = ({ placeholderFacet, valueFacet }) => {
+const WrapperComponent = () => {
 	const [facetA, setFacetA] = useFacetState('A')
 	const [facetB, setFacetB] = useFacetState('B')
 
 	const groupedFacet = useFacetMap(
 		(a, b) => [a, b]
```

Comment:
This example has an error that I'm not sure how to solve https://jolly-stone-0d2f00e03-58.westeurope.azurestaticapps.net/docs/api/hooks#usefacetmap